src/main/kotlin/us/ihmc/build/IHMCBuildExtension.kt

```diff
@@ -117,7 +117,6 @@ open class IHMCBuildExtension(val project: Project)
    fun configureDependencyResolution()
    {
       declareMavenCentral()
-      repository("https://clojars.org/repo/")
```

Comment:
How can you be sure this doesn't break anything for anybody? Is it that useful to remove?

Reply:
I would say we should be arguing why to keep it, not why to remove it. I don't see any reason anyone would be using it. Before I make any release of ihmc-build, I would double check that, at the very least, all of our repos resolve dependencies correctly and things are copesthetic.

Reply:
I guess it sketchy clojure libraries mostly, actually